bse/bseapi.idl

```diff
@@ -1135,6 +1135,11 @@ interface MidiSynth : SNet {
                                 STANDARD ":scale",
                                 1, 256, 1, 16);
   };
+  group _("Adjustments") {
+    float64 volume_f = Range (_("Master [float]"), _("Master volume as factor"),
+                              STORAGE,
+                              0, 15.8489319246111 /* +24dB */, 0.1, 1.0);
```

Comment:
15.84... used to be calculated as a factor from MAX_dB, if we every adjust MAX-dB in the IDL file, this needs to be adjusted as well, so it shouldn't be hardcoded.
I.e. just add another constant next to the MAX_dB constant definition and use that for this registration, so it's obvious that both have to be adapted if we touch the constants in the future.